src/main/java/seedu/application/model/ModelManager.java

```diff
@@ -28,7 +28,6 @@ public class ModelManager implements Model {
     private final FilteredList<Application> filteredApplications;
     private final SortedList<Application> sortedFilteredApplications;
     private final FilteredList<Application> applicationsWithInterview;
-    private final ObservableList<Application> applicationsWithUpcomingInterviews;
```

Comment:
Basically no longer keeping this value but returning the filtered list every time `remind` is called